### PR TITLE
Add a summary table to therapeutics

### LINKED ISSUE #861: Add a summary table for all interventions
**Is your feature request related to a problem? Please describe.**
@SiminaB suggested adding a table that summarizes the status of the treatments we review. This sounds like an excellent idea to me!

> I'm thinking having a column for treatment, one for approval status (in US and other country, whether full or emergency approval), and one for status of evidence (eg from computational predictions only, from in vitro studies only, from non-randomized studies, single randomized study, different randomized studies disagree - eg for remdesivir, multiple randomized studies agreeing on benefit etc)

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

This is a work in progress to add a summary table to the conclusions section of the therapeutics paper. I used information from these sources to assess FDA approval:
- https://www.fda.gov/media/136832/download
- https://www.ohsu.edu/sites/default/files/2020-09/Travel-free%20CME%20-%20FDA%20approval.pdf
- https://covid.idea.medicine.uw.edu/page/treatment/drugs/tocilizumab

In the "files changed" tab, if you click the button that looks like a sheet of paper, you can see it rendered as a table instead of in the raw format.

@nilswellhausen would you mind updating this this weekend once you go through the monoclonal antibodies? I added all the ones from the FDA list!

### Related issues

closes #861 

### Suggested reviewers (optional)

@SiminaB 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by SiminaB** [comment]
Didn't realize it wasn't ready for review yet so I will leave my suggested changes below. Looks really good!

TCZ edits are based on summary at https://emcrit.org/pulmcrit/tocilizumab/. We will need to look at those papers individually though.

| Treatment | Category | FDA Status | Evidence Available | Suggested Effectiveness |
|-----------|----------|-----------------|--------------------|-------------------------|
| Favipiravir | Small molecule, antiviral, nucleoside analog | None | RCT | Not supported: RCTs do not show significant improvements for individuals taking this treatment, good safety profile | 
| Remdesivir | Small molecule, antiviral, adenosine analog | Approved (and EUA for combination with baricitinib) | RCT | Conflicting evidence from large WHO-led Solidarity trial vs US-focused RCT and other studies |
| N3 | Small molecule, protease inhibitor | None | Computational prediction, _in vitro_ studies | Unknown |
| ARBs & ACEIs | Small molecule, broad spectrum | None | Observational studies, RCTs ongoing | Not supported: Observational study retracted, RCTs ongoing |
| HCQ/CQ | Small molecule, broad spectrum | None | RCT  | Not supported, possibly harmful: Non-blinded RCTs showed no improvement over SOC, safety profile may be problematic |
| Dexamethasone | Small molecule, broad spectrum | Used off-label | RCT | Supported: RCT shows improved outcomes over SOC, especially in severe cases such as CRS |
| Tocilizumab | Biologic, monoclonal antibody | Approved for CRS resulting from CAR-T therapy | RCT | Mixed results from RCTs: It appears that TCZ may work well in combination with dexamethasone in severe cases, but not as monotherapy |
| Possibly other antibodies? | 
| Casirivimab and imdevimab | Biologic, monoclonal antibodies | EUA | | | |
| Bamlanivimab and etesevimab | Biologic, monoclonal antibodies | EUA | | | |
| SNG001 | Biologic, interferon | None | RCT | Likely not supported |

**Comment by nilswellhausen** [comment]
@rando2. Yes I can update that with the antibodies. There are actually tons of them in trials, do you want me to restrict it to the most promising ones to not blow that out of proportion?

**Comment by rando2** [comment]
@nilswellhausen My gut would be to focus on the ones that have EUAs but also mention that there are more? I definitely defer to your judgment!!! I just stuck these ones on the list since I was going through a list of EUAs!

**Comment by rando2** [comment]
@SiminaB, thank you so much for filling this in with so much detail!!! It looks amazing. 
The RCT I was thinking of for TCZ is the RECOVERY trial in #864 and the RCTs for ARBs were recently added in #860 -- but that is my main addition! I'll work on incorporating it!

File: content/20.pharmaceuticals.md [hunk 1]
```diff
@@ -845,7 +847,23 @@ Over the longer term, this approach and others may lead to the development of no
 
 #### Conclusions <!--I’m not sure we need this section. We’ve stated these things before in this manuscript and the section reads as how useful it’s been to learn from SARS-CoV-2 and MERS-CoV – it has been useful, but I don’t think we need that as the overall conclusion…-->
 
-Due to the large number of clinical trials currently under examination (Figure @fig:ebm-trials), not all candidates are examined here.
+| Treatment | Category | FDA Status | Evidence Available | Suggested Effectiveness |
+|-----------|----------|-----------------|--------------------|-------------------------|
+| Favipiravir | Small molecule, antiviral, nucleoside analog | None | RCT | Not supported | 
+| Remdesivir | Small molecule, antiviral, adenosine analog | Approved (and EUA for combination with baricitinib) | RCT | Conflicting evidence from Solidarity vs other trials |
+| N3 | Small molecule, protease inhibitor | None | Computational prediction | Unknown |
+| ARBs & ACEIs | Small molecule, broad spectrum | None | RCT | Not supported |
```
Comment (rando2) [inline content/20.pharmaceuticals.md:855]:
```suggestion
| ARBs & ACEIs | Small molecule, broad spectrum | None | Observational studies and some RCTs | Not supported: Observational study retracted, RCTs suggest no association |
```

File: content/20.pharmaceuticals.md [hunk 2]
```diff
@@ -845,7 +847,23 @@ Over the longer term, this approach and others may lead to the development of no
...
+| HCQ/CQ | Small molecule, broad spectrum | None | RCT  | Not supported, possibly harmful |
```
Comment (rando2) [inline content/20.pharmaceuticals.md:856]:
```suggestion
| HCQ/CQ | Small molecule, broad spectrum | None | RCT  | Not supported, possibly harmful: Non-blinded RCTs showed no improvement over SOC, safety profile may be problematic |
```

File: content/20.pharmaceuticals.md [hunk 3]
```diff
@@ -845,7 +847,23 @@ Over the longer term, this approach and others may lead to the development of no
...
+| Dexamethasone | Small molecule, broad spectrum | Used off-label | RCT | Supported |
```
Comment (rando2) [inline content/20.pharmaceuticals.md:857]:
```suggestion
| Dexamethasone | Small molecule, broad spectrum | Used off-label | RCT | Supported: RCT shows improved outcomes over SOC, especially in severe cases such as CRS |
```

File: content/20.pharmaceuticals.md [hunk 4]
```diff
@@ -845,7 +847,23 @@ Over the longer term, this approach and others may lead to the development of no
...
+| Tocilizumab | Biologic, monoclonal antibody | Approved for CRS | RCT | Supported |
```
Comment (rando2) [inline content/20.pharmaceuticals.md:858]:
```suggestion
| Tocilizumab | Biologic, monoclonal antibody | Approved for CRS resulting from CAR-T therapy | RCT | Mixed results from RCTs: It appears that TCZ may work well in combination with dexamethasone in severe cases, but not as monotherapy |
```

File: content/20.pharmaceuticals.md [hunk 5]
```diff
@@ -845,7 +847,23 @@ Over the longer term, this approach and others may lead to the development of no
...
+| Possibly other antibodies? | 
+| Casirivimab and imdevimab | Biologic, monoclonal antibodies | EUA | | | |
+| Bamlanivimab and etesevimab | Biologic, monoclonal antibodies | EUA | | | |
+| SNG001 | Biologic, interferon | None | RCT | Likely not supported |
```
Comment (rando2) [inline content/20.pharmaceuticals.md:862]:
```suggestion
| SNG001 | Biologic, interferon | None | RCT | Mixed results: support from initial RCT but no effect found in WHO's Solidarity trial |
```

**Review by rando2 — COMMENTED** [review]
Update table in line with @SiminaB's additions and some additional info

**Comment by rando2** [comment]
@SiminaB if this looks good, I guess we should just add this version to the manuscript so it's easier for @nilswellhausen to update? I took it out of draft mode!

**Review by SiminaB — APPROVED** [review]

**Pull request merged**